### PR TITLE
chore: apply clippy suggestions.

### PR DESCRIPTION
**Pull request by FabianLars**

### CODE REVIEW

**Comment by FabianLars** [comment]
stupid os specific code 😠

**Comment by FabianLars** [comment]
it wasn't os specific code 😂

File: crates/tauri-cli/src/interface/rust/desktop.rs [hunk 1]
```diff
@@ -440,7 +440,7 @@ mod terminal {
       // INVALID_HANDLE_VALUE. Use an alternate method which works
       // in that case as well.
       let h = CreateFileA(
-        "CONOUT$\0".as_ptr() as PCSTR,
+        c"CONOUT$".as_ptr() as PCSTR,
```
Comment (lucasfernog) [inline crates/tauri-cli/src/interface/rust/desktop.rs:443]: TIL

**Review by lucasfernog — APPROVED** [review]

File: crates/tauri-cli/src/interface/rust/desktop.rs (hunk 1 above)
Comment (FabianLars) [reply crates/tauri-cli/src/interface/rust/desktop.rs:443]: Same, almost thought clippy is trolling me

**Pull request merged**